package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "fast-copy": "^2.1.0",
+    "lodash": "^4.17.21",
```

Comment:
Did you try [`lodash-es`](https://preview.npmjs.com/package/lodash-es) for better tree shaking support?